ShanghaiBloodSports/Assets/Scripts/Trinkets/BaseTrinkets/DeathTrinket.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Assets.Scripts.Events;
+using UnityEngine;
+
+namespace Assets.Scripts.Trinkets.BaseTrinkets
+{
+    class DeathTrinket : TrinketBase
+    {
+        private Handler handler;
+        private Character owner;
+
+        public override string Name => "Frailty of Man";
+        public override string Description => "People die when they are killed.";
```

Comment:
Just because you're correct doesn't mean you're right

Reply:
Just wait until we add the trinket that says "You don't die when you are killed"